apps/web/components/team/MemberChangeRoleModal.tsx

```diff
@@ -37,6 +38,23 @@ export default function MemberChangeRoleModal(props: {
       memberId: props.memberId,
       role,
     });
+
+    if (role === "OWNER") {
+      const members = props.team["members"];
+      let owner;
+
+      for (let i = 0; i < members.length; i++) {
+        if (members[i].role === "OWNER") {
+          owner = members[i].id;
+        }
+      }
```

Comment:
this for can be changed to **find** 
`const owner = members.find((item) => item.role === MembershipRole.OWNER); 
`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find